src/main.rb

```diff
@@ -0,0 +1,50 @@
+require './app'
+
+class Main
+  def self.home_page
+    puts 'Please type a valid number: '
+
+    @content = {
+      '1' => 'List all books',
+      '2' => 'List all people',
+      '3' => 'Create a person (teacher or student)',
+      '4' => 'Create a book',
+      '5' => 'Create a rental',
+      '6' => 'List all rentals for a given person id',
+      '7' => 'Exit'
+    }
+    @content.each { |index, string| puts "#{index} - #{string}" }
+
+    Integer(gets.chomp)
+  end
+
+  app = App.new
+
+  loop do
+    case home_page
+    when 1
+      app.all_booklist
+    when 2
+      app.all_peoplelist
+    when 3
```

Comment:
- [ ] After fixing the above error :point_up: , your app was working perfectly :+1: and I was really happy but unfortunately that I couldn't list a book and people created. 

> **Hints:**
  **Once more let's kindly pay attention to details. You are calling a method on the `app` instance that does not exist at all :angry:. Get into the class **App** and verify if this method(`all_booklist`) exists. 
I think there's no such method but rather we have the method `list_all_books`**

That goes the same as option 2(**all_peoplelist**). This method does not exist, rather you created the method `list_all_people` hence you should be referencing **app.list_all_people**